templates/typescript_gapic/_locations.njk

```diff
@@ -0,0 +1,82 @@
+{%- macro locationServiceMethods() -%}
+{#-
+  This part will be added into src/version/client.ts.
+  If the --service-yaml contains "google.iam.v1.IAMPolicy", it means the client requires IAM methods [setIamPolicy, getIamPolicy, testIamPerssion].
+  IamClient is created for the client in the constructor.
+  [setIamPolicy, getIamPolicy, testIamPermission] methods are created which is calling the corresponding methods from IamClient in google-gax.
```

Comment:
nit: Should we use the location information here. :)

Reply:
great catch! thank you, updated.